internal/pkg/api/repository/config/mongo.go

```diff
@@ -31,14 +31,20 @@ func (r *MongoRepository) GetUniqueClusterId(ctx context.Context) (clusterId str
 
 	// generate new cluster Id and save if there is not already
 	if config.ClusterId == "" {
-		config.ClusterId = fmt.Sprintf("cluster%s", analytics.MachineID())
+		config.ClusterId = fmt.Sprintf("cluster%s", telemetry.MachineID())
 		err := r.Upsert(ctx, config)
 		return config.ClusterId, err
 	}
 
 	return config.ClusterId, nil
 }
 
+func (r *MongoRepository) GetTelemetryEnabled(ctx context.Context) (ok bool, err error) {
```

Comment:
Shoud we store it in mongodb? Might be just use ConfigMap?

Reply:
Yeah. Was thinking about it but need to research problems with access from several nodes at once first. Will add task about it